amazon_cloud_code_generator/data/plugins/module_utils/core.py

```diff
@@ -99,18 +99,11 @@ def wait_until_resource_request_success(self, request_token: str):
                 RequestToken=request_token,
                 WaiterConfig=self._waiter_config,
             )
-        except botocore.exceptions.WaiterError as e:
+        except Exception as e:
```

Comment:
We should leave this as `botocore.exceptions.WaiterError` because other exceptions won't have an `e.last_response` attribute.